File: Practicas Samurai/0004 - par_impar.js
//Escribir un programa que pida al usuario un número entero y muestre por pantalla si es par o impar.
let numero = "a"
while (numero < 0 || isNaN(numero)) {
    numero = Number(prompt('Ingrese una edad válida: '))
}
let paridad = numero % 2
if (paridad !== 0){
    alert("No es par")
} else {
    alert("es par")
}

File: Practicas Samurai/terminados/0004 - par_impar.py
#Escribir un programa que pida al usuario un número entero y muestre por pantalla si es par o impar.

numero = -1
while True:
    try:
        numero = int(input('Ingrese un numeró: '))
        break
    except ValueError:
        print("Error, ingreso no válido.")

paridad = numero % 2
if paridad != 0:
    print("No es par")
else:
    print("es par")